sequencer.c

```diff
@@ -154,7 +154,7 @@ static void free_message(struct commit *commit, struct commit_message *msg)
 	unuse_commit_buffer(commit, msg->message);
 }
 
-static void print_advice(int show_hint, struct replay_opts *opts)
+static void print_advice(int show_hint, struct replay_opts *opts, int last_commit)
```

Comment:
This is probably misnamed. You seem to be using it as a boolean that lets this function tell if this is the last commit, but it sounds as if you are referring to nth commit in a sequence and passing that number to this function.

Perhaps rename it "is_last" or something?